tests/Unit/Evolution/DiscontinuousGalerkin/Test_BoundaryCorrectionsHelper.cpp

```diff
@@ -82,7 +112,13 @@ struct Correction {
       const boost::optional<tnsr::I<DataVector, Dim, Frame::Inertial>>&
       /*mesh_velocity*/,
       const boost::optional<Scalar<DataVector>>& normal_dot_mesh_velocity,
-      const double volume_double) const noexcept {
+      const VolumeDoubleType volume_double_in) const noexcept {
```

Comment:
I'm sure this will be clearer in the documentation for the specific boundary corrections when you start putting them in, but for my understanding, can you explain the role of this `volume_double` in these packaging functions?

Reply:
It's just to test that quantities from the volume (i.e. DataBox) can be passed in. A practical example is needing the equation of state (which can't be projected onto the boundary) in order to calculate the speed of sound, which is needed to compute the characteristic speeds. The Newtonian Euler system uses this pattern, for example.